tests/model/ErrorPageTest.php

```diff
@@ -64,13 +64,18 @@ public function testBehaviourOfShowInMenuAndShowInSearchFlags() {
 	public function testBehaviourOf403() {
 		$page = $this->objFromFixture('ErrorPage', '403');
 		$page->publish('Stage', 'Live');
-		
-		$response = $this->get($page->RelativeLink());
-		
-		$this->assertEquals($response->getStatusCode(), '403');
-		$this->assertNotNull($response->getBody(), 'We have body text from the error page');
+
+		try {
+			$controller = singleton('ContentController');
+			$controller->httpError(403);
```

Comment:
After this line put a `$this->fail('Expected exception to be thrown')` as well. :)